src/httpStatus/HttpSuccessStatus.ts

```diff
@@ -1,13 +1,95 @@
+/**
+ * @enum Http Successful Responses
+ */
 enum HttpSuccessStatus {
+    /**
+     * 200 OK - The request has succeeded. The meaning of the success depends on the HTTP method:
+     * GET: The resource has been fetched and is transmitted in the message body.
+     * HEAD: The entity headers are in the message body.
+     * PUT or POST: The resource describing the result
+     * of the action is transmitted in the message body.
+     * TRACE: The message body contains the request message as received by the server.
+     * For more details check out
+     * { @link https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200 }
+     * @type {number}
+     */
     Ok = 200,
```

Comment:
Can we make it more cleaner like this.

    /**
     * ### 200 OK
     * #### The request has succeeded. 
     * 
     * 
     * The meaning of the success depends on the HTTP method:  
     * 
     * **GET:** The resource has been fetched and is transmitted in the message body.  
     * **HEAD:** The entity headers are in the message body.  
     * **PUT or POST:** The resource describing the result.  
     * of the action is transmitted in the message body.  
     * **TRACE:** The message body contains the request message as received by the server.
     * 
     * {@link https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200|Read more}
     * 
     */
    Ok = 200,

results in this
![image](https://user-images.githubusercontent.com/16626680/117865451-9b571e00-b2b3-11eb-9038-fe4424b42796.png)

To insert a new line in the comments
Add two spaces before the end of the line, and then select **Enter**.(space, space, Enter)
A space gets added in between paragraphs.